zipkin-lens/package.json

```diff
@@ -70,12 +65,20 @@
     "react-redux": "^5.0.7",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
+    "react-scripts": "^2.1.3",
     "react-select": "^2.0.0",
     "react-table": "^6.8.6",
     "react-transition-group": "^2.4.0",
     "redux": "^4.0.0",
     "redux-thunk": "^2.3.0",
     "shortid": "^2.2.14",
     "vizceral-react": "^4.6.5"
-  }
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ],
+  "proxy": "http://localhost:9411"
```

Comment:
This may be a very basic question, but what are `browserlist` and `proxy` settings used for?